app/views/layouts/megatron/application.html.slim

```diff
@@ -8,10 +8,12 @@ html lang="en"
     meta name="viewport" content="width=device-width,initial-scale=1"
     meta charset="utf-8"
     meta http-equiv="X-UA-Compatible" content="IE=edge"
+    meta name="bugsnag" data-api-key="#{ENV['BUGSNAG_API_KEY']}" data-release-stage="#{Rails.env}"
```

Comment:
Good call on the API Key EVN name. Rails.env will be development when you're running this locally. That's going to be important to avoid logging to Bugsnag while working locally. I don't know if there's some way to route stuff differently in staging though.